lib/modules/manager/gleam/update-locked.ts

```diff
@@ -0,0 +1,25 @@
+import { logger } from '../../../logger';
+import type { UpdateLockedConfig, UpdateLockedResult } from '../types';
+import { extractLockFileContentVersions } from './locked-version';
+
+export function updateLockedDependency(
+  config: UpdateLockedConfig,
+): UpdateLockedResult {
+  const { depName, currentVersion, newVersion, lockFile, lockFileContent } =
+    config;
+  logger.debug(
+    `gleam.updateLockedDependency: ${depName}@${currentVersion} -> ${newVersion} [${lockFile}]`,
```

Comment:
are other managers logging here too? if not reduce to trace level

Reply:
There are some precedents for `debug` being used here instead of `trace`, such as here in [Cargo](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/cargo/update-locked.ts#L10) and here in [Poety](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/poetry/update-locked.ts#L11). I am open to reducing this to `trace` however, if you think that is better.

Reply:
yes, reduce, users can use log level remap if they need to troubleshoot something

Reply:
since we're likely removing this function, i'll "resolve" this without change for now.